## 01_no_usr-local.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
## Description: Don't install under /usr/local
Index: Turnin-NG/setup.py
===================================================================
--- Turnin-NG.orig/setup.py	2009-08-04 12:54:49.427707124 -0400
+++ Turnin-NG/setup.py	2009-08-04 13:02:58.179716514 -0400
@@ -12,5 +12,5 @@
       scripts=['src/bin/project', 'src/bin/turnin'],
       packages=['turninng'],
       package_dir={'turninng':'src/turninng'},
-      data_files=[('/usr/local/share/man/man1/', ['doc/turnin.1', 'doc/project.1'])]
+      data_files=[('/usr/share/man/man1/', ['doc/turnin.1', 'doc/project.1'])]
 )
